Sources/TensorFlow/Loss.swift

```diff
@@ -17,23 +17,42 @@
 /// - Parameters:
 ///   - predicted: Predicted outputs from a neural network.
 ///   - expected: Expected values, i.e. targets, that correspond to the correct output.
+///   - reduction: Reduction to apply on the computed element-wise loss values.
```

Comment:
@eaplatanios Just a suggestion, If there aren't too many reductions available, could we mention what they are ? or do swift reductions necessarily have to be given in the ` $0."function"` format?

Reply:
You can pass any closure you want as a reduction. In principle this is more of a "postprocessing" function but I believe calling it "reduction" is fine as that is the intended use and it's mainly added to maintain source compatibility with existing code.